certified-connectors/PDF4me Connect/apiDefinition.swagger.json

```diff
@@ -2461,6 +2461,315 @@
           }
         ]
       }
+    },
```

Comment:
PR should contain only files for one connector. Please either exclude this file from the PR or submit new PR with only files related to the Converter connector.

Reply:
Hi @stsiek, I hope you are doing well. it looks like you are updating "PDF4me Connect" as well in this PR. I suggest you close this PR and resubmit a new PR with the appropriate connector files only.
Please let us know if you have any questions.
Thank you very much